src/components/textarea.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(app) {
             }
             return 'formio/components/texteditor.html';
           }
+          else if (!$scope.readOnly && $scope.component.components) {
```

Comment:
I am confused by the condition when the scripteditor is triggered. This tells me that any component that has components within them will trigger the scripteditor?  What is the use case for this? Can we not introduce a new configuration that, if present, would turn a textarea into a scripteditor?